drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -28,6 +28,8 @@ class Report < SimpleReports::ReportInstance
 
     after_initialize :filter
 
+    SPM_PROJECT_TYPES = [:es, :so, :sh, :th, :ph].freeze
```

Comment:
This is tiny and could wait for a future pass, but maybe this could move to `GrdaWarehouse::Hud::Project.SPM_PROJECT_TYPE` just to centralize it?